modules/alma/lib/AlmaClient/AlmaClient.class.php

```diff
@@ -980,6 +988,37 @@ public function remove_message_service($borr_card, $pin_code, $method, $type) {
     return TRUE;
   }
 
+  /**
+   * Create new user af alma.
+   *
+   * @param $cpr
+   * @param $pin_code
+   * @param $name
+   * @param $mail
+   * @param $branch
```

Comment:
Please add more elaborate documentation. Argument types and descriptions would be good.